webapp/resources/templates/fragments/pwdupdateform.html

```diff
@@ -7,7 +7,12 @@
 </script>
 
 <div class="alert alert-info">
-    <h2 th:utext="#{screen.mustchangepass.heading}"/>
+    <div th:if="${execInfo.processedTemplateName} == 'casExpiredPassView'">
```

Comment:
That would work, but ideally the backend would send a flag, but creating a variable in the page would be the easiest to implement.  Just want to make it as easy as possible to update in the future if needed.  Thanks!